lib/compiler/include/compiler/cost_estimate.h

```diff
@@ -0,0 +1,21 @@
+
+#ifndef _FLEXFLOW_COMPILER_COST_ESTIMATE_H
+#define _FLEXFLOW_COMPILER_COST_ESTIMATE_H
+
+#include "op-attrs/operator_attrs.h"
+#include "pcg/machine_view.h"
+
+namespace FlexFlow {
+
+struct ICostEstimator {
```

Comment:
Add `is_rc_copy_virtual_compliant` assertion

Reply:
And `CostEstimator` wrapper class if it does not yet exist to enable standard value semantics (e.g., copies, moves, etc.)